monai/transforms/post/array.py

```diff
@@ -46,3 +48,72 @@ def __call__(self, img, to_onehot=None, num_classes=None):
             outputs.append(img[:, i : i + 1])
 
         return outputs
+
+
+class AddActivations(Transform):
+    """
+    Add activation operations to the model output, typically `Sigmoid` or `Softmax`.
+
+    Args:
+        add_sigmoid (bool): whether to add sigmoid function to model output before transform.
+        add_softmax (bool): whether to add softmax function to model output before transform.
+        other (Callable): callable function to execute other activation layers, for example:
+            `other = lambda x: torch.tanh(x)`
+
+    """
+
+    def __init__(self, add_sigmoid=False, add_softmax=False, other=None):
+        self.add_sigmoid = add_sigmoid
+        self.add_softmax = add_softmax
+        self.other = other
+
+    def __call__(self, img, add_sigmoid=None, add_softmax=None, other=None):
+        if add_sigmoid is True and add_softmax is True:
+            raise ValueError("add_sigmoid=True and add_softmax=True are not compatible.")
+        if self.add_sigmoid if add_sigmoid is None else add_sigmoid:
```

Comment:
`self.add_sigmoid if add_sigmoid is None else add_sigmoid` could be simplified as
`add_sigmoid or self.add_sigmoid`

Reply:
updated in the latest patch.
Thanks.